csrankings-9.csv

```diff
@@ -1225,6 +1225,7 @@ Xue Chen 0001,George Mason University,https://sites.google.com/view/xuechencs/ho
 Xue Dong Yang,University of Regina,http://www.cs.uregina.ca/People/Profiles/XueDongYang.html,NOSCHOLARPAGE
 Xue Li 0001,University of Queensland,http://staff.itee.uq.edu.au/xue,3eIIFWIAAAAJ
 Xue Lin,Northeastern University,https://web.northeastern.edu/xuelin,p87KNLIAAAAJ
+Xue-Cheng Tai,Hong Kong Baptist University,http://www.math.hkbu.edu.hk/~xuechengtai/HKBU.html,XOf_txgAAAAJ

```

Comment:
Math, not CS.

Reply:
While being another department, his research area is heavily focused on image processing and computer vision.

Reply:
I have removed this entry.